Sign-up.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+import 'dart:convert';
+import 'Welcome.dart';
+import 'home.dart';

```

Comment:
this kind of import causes circular imports or dependencies. Please try to avoid it somehow, you can create a third entity which is the master that has everything imported in.